components/centraldashboard/public/components/main-page.pug

```diff
@@ -1,44 +1,40 @@
 app-drawer-layout.flex(narrow='{{narrowMode}}')
+    app-location(route='{{route}}')
+    app-route(route='{{route}}', pattern='/:page', data='{{routeData}}', tail='{{subRouteData}}')
+
+
     app-drawer#MainDrawer(slot='drawer')
         figure#Logo
             img(alt='Kubeflow Logo', src='assets/kf-logo_64px.svg')
             figcaption Kubeflow
-        iron-selector#SidebarSelector(selected='{{sidebarItemIndex}}')
-            template(is='dom-repeat', items='[[links]]')
-                paper-item.menu-item(on-click='openLink') [[item.text]]
-                template(is='dom-if', if='[[item.hasDivider]]')
-                    aside.divider
+        iron-selector(selected='{{sidebarItemIndex}}')
+            a(href='/', tabindex='-1')
+                paper-item.menu-item Home
+                aside.divider
+            template(is='dom-repeat', items='[[menuLinks]]')
+                a(href$='[[item.href]]', on-click='openInIframe', tabindex='-1')
+                    paper-item.menu-item [[item.text]]
         aside.flex
         footer.footer
             section.privacy Privacy
-            section.build build version 
-                span(title="Build: v[[buildVersion]] | Dashboard: v[[dashVersion]]") v[[buildVersion]]
+            section.build build version
+                span(title="Build: v[[buildVersion]] | Dashboard: v[[dashVersion]]")  v[[buildVersion]]
```

Comment:
Revert change on this line

Reply:
My editor is set to trim trailing whitespace which would be a common setting many users would have. I added the `&nbsp` to be explicit.